ocrd_segment/extract_lines.py

```diff
@@ -248,3 +237,42 @@ def process(self):
                     i += 1
 
             workbook.close()
+
+    def _get_presentation_image(self, input_file, library_convention):
+        if library_convention == 'slub':
+            return self._get_presentation_image_slub(input_file)
+        elif library_convention == 'sbb':
+            return self._get_presentation_image_sbb(input_file)
+        raise NotImplementedError("Unsupported library convention '%s'" % library_convention)
```

Comment:
I suggest we allow setting _no_ convention, too. After all, clickable images are just a comfort feature.

Reply:
True, I'll add that.

Reply:
b876574 the default is now none, to just add an empty string as the URL. If that breaks your scripts, I can change default back to `slub`.

Reply:
It's a parameter now, so no problem.

I just wonder whether (after merging this) I should rebase that branch on master, stripping out the `maskrcnn-cli` merge. After all, one can always merge branches in a local deployment when both features are needed, but not everyone wants all those extra dependencies (at least for now)...

Reply:
I had no problem installing the dependencies, that the `maskrcnn` clones the github repository on every install is a bit annoying.

Do you want to do the cherry-picking or shall I?

Reply:
> that the `maskrcnn` clones the github repository on every install is a bit annoying.

Yes. It's because of the `git+https` spec in my requirements – I have to use my own fork, but don't want to publish under PyPI (yet). I'd have to come up with a good alternative package name for that, but since the original is unmaintained now...

I guess I could try adding a fixed git tag to the URL, but IIUC the actual reason for re-installation each time is that I don't use `--editable` ...

> Do you want to do the cherry-picking or shall I?

No thanks, IIRC I had some merge conflicts earlier – this I have to fiddle together by myself.

Reply:
> I guess I could try adding a fixed git tag to the URL

Don't bother, it will still clone the repo to get that tag in my experience (it's a limitation of git).

> No thanks, IIRC I had some merge conflicts earlier – this I have to fiddle together by myself.

I dodged a bullet there :) Thanks!